code/imageloading.py

```diff
@@ -6,6 +6,7 @@
 
 '''
 import scipy.io as sio
+import matplotlib.pyplot as plt
 
 def load_image(path):
```

Comment:
I think the purpose of this function should be to load the matlab file and return a numpy array

a separate function should load this numpy array and display it